Documentation/ceph-advanced-configuration.md

```diff
@@ -590,3 +591,55 @@ ceph osd pool get replicapool crush_rule
 If the cluster's health was `HEALTH_OK` when we performed this change, immediately, the new rule is applied to the cluster transparently without service disruption.
 
 Exactly the same approach can be used to change from `host` back to `osd`.
+
+## Auto Expansion of OSDs
```

Comment:
At the top of this doc there is a table of contents, if you could add a link to this section.

Reply:
* [Auto Expansion of OSDs](#auto-expansion-of-OSDs), I have changed like this.